test/kubemark/start-kubemark-master.sh

```diff
@@ -19,6 +19,11 @@
 EVENT_STORE_IP=$1
 EVENT_STORE_URL="http://${EVENT_STORE_IP}:4002"
 NUM_NODES=$2
+TEST_ETCD_VERSION=$3
+if [[ -z "${TEST_ETCD_VERSION}" ]]; then
```

Comment:
TEST_ETCCD_VERSION is initialized in the above line, so it's not needed to default it.